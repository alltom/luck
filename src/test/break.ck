repeat(5) {
    <<< "should see this once" >>>;
    break;
}
<<< "done" >>>;
