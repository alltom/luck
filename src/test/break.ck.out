should see this once
done
